Makefile.docker

```diff
@@ -11,19 +11,18 @@ DOCKERFILE_DIR     ?= ./
 DOCKER_REGISTRY    ?= docker.io
 DOCKER_ORG         ?= $(USER)
 DOCKER_TAG         ?= latest
-DOCKER_BUILD_TAG   ?= $(DOCKER_TAG)
 DOCKER_VERSION_ARG ?= latest
 
 all: docker_build docker_push
 
 docker_build:
-	echo "Building Docker image ..."
-	docker build --build-arg version=$(DOCKER_VERSION_ARG) -t strimzi/$(PROJECT_NAME):$(DOCKER_TAG) $(DOCKERFILE_DIR)
+	# Build Docker image ...
+	docker build --build-arg version=$(DOCKER_VERSION_ARG) -t strimzi/$(PROJECT_NAME):latest $(DOCKERFILE_DIR)
```

Comment:
We should be careful when merging between this and the PR which removes the version build argument so that we don't reintroduce it.